doc/src/odin/user_guide/verilog_support/c_functions.md

```diff
@@ -0,0 +1,12 @@
+## C Functions support
```

Comment:
is this gonna generate separate pages? IDK if i would prefer all the tables in one page

Reply:
They're all on seperate pages, I just pushed it so I could see what it looks like on github.

Reply:
Do you like how theres a table of contents and then you can select what you want for user guide?

Reply:
I do :)